redbeams/src/main/java/com/sequenceiq/redbeams/domain/stack/DBStack.java

```diff
@@ -33,6 +33,10 @@
     @SequenceGenerator(name = "dbstack_generator", sequenceName = "dbstack_id_seq", allocationSize = 1)
     private Long id;
 
+    @Convert(converter = CrnConverter.class)
+    @Column(nullable = false)
+    private Crn resourceCrn;
```

Comment:
What is the reason for Crn object? We add more unnecessary conversation to the process.

Reply:
Yes, string would be easier. 
This way, however, 
1) you can be sure that the stored string can be mapped to a crn, and
2) the type of the variable tells you what this thing is (crn), not the name.